### DISCUSSION TITLE
Date and Time

### DISCUSSION THREAD

**Discussion by 0x07cc** [kovidgoyal/kitty#3394] [Q&A]
Hi. I'm trying to show date and time at the bottom of my screen.
Is there a simple way to do it?

I've tried:

1. Setting time in the tab name
![tab](https://user-images.githubusercontent.com/66702016/111087620-83b12280-851a-11eb-99ab-92e1e11f4d28.png)

2. Setting time in a window
![window](https://user-images.githubusercontent.com/66702016/111087621-8449b900-851a-11eb-937b-70620779368b.png)

This a test to update time with a python script:
![tabs+python](https://user-images.githubusercontent.com/66702016/111088150-b4468b80-851d-11eb-8c1a-72321fc7895a.png)

Thanks!

**Comment by kovidgoyal** [comment]
There is no builtin way to do this. Someday if I implement #2391 it will be trivial to do.

**Reply by 0x07cc** [reply]
Ok thank you.
